tl/features/string_similarity.py

```diff
@@ -76,6 +76,7 @@ def get_similarity_score(self):
         for _, each_row in self.df.iterrows():
             for each_similarity_unit in self.similarity_units:
                 # the output column name, should be the type + config for this similarity calculation unit
+                # TODO: naming is weird
```

Comment:
what are your changes in this file?

Reply:
`string-similarity` actually doesn't have a -o option to name output column

Reply:
I added output column option in my latest commit